packages/common/tests/supportedChainId.spec.ts

```diff
@@ -0,0 +1,15 @@
+import tape from 'tape'
+import Common from '../src/'
+import { BN } from 'ethereumjs-util'
+
+tape('[Common]: isSupportedChainId static method', function (t: tape.Test) {
+  t.test('Should return correct response', function (st: tape.Test) {
+    st.equal(Common.isSupportedChainId(new BN(1)), true, 'should return true for supported chainId')
+    st.equal(
+      Common.isSupportedChainId(new BN(1234)),
+      false,
+      'should return false for unsupported chainId'
+    )
+    st.end()
+  })
+})
```

Comment:
Thanks for the PR! 😄 

Two things here:

1. We currently do not organize the tests in the libraries on that granularity level of having one file per method and so this test file would jump somewhat out of the line, can you please add to one of the existing files? I guess `chains.spec.ts` should be a good candidate. Not that we can't reorg at some PIT if we see the need, but these kind of things should generally be discussed before.
2. Can you also add a positive test case here for a chain ID existing?

Reply:
@holgerd77 Thanks for the review!

Re: 1, 👍. I wasn't sure what existing file it should go in so I made a new one, but your point makes sense. I'll move the test into `chains.spec.ts`.

Re: 2, I think I have that already here (https://github.com/ethereumjs/ethereumjs-monorepo/pull/1281/files#diff-00e881872be66da72fe4e5df60785751bb9b0fce068df279a7fd63b9dcdd0423R7), unless I'm not understanding? Actually now that I look at it, it is a little awkward having the top-level test be `should return the correct response`. I can break them out into separate tests.

Reply:
Also, I had a question unrelated to my code changes, just something I noticed. There's a file in `packages/common/tests` named `customChains.ts`. Should the file name be `customChains.spec.ts`?

Reply:
@emersonmacro oh yes, good catch. 😄  We have a collecting issue open here where we collect some of the smaller fixes to then be worked on in batch (or along some other PRs) https://github.com/ethereumjs/ethereumjs-monorepo/issues/905, have added there so that it won't get lost.